resources/install.ps1

```diff
@@ -24,18 +24,18 @@ if (-not (Test-Path $sp_dot_dir)) {
 }
 
 Write-Host "Downloading..." -ForegroundColor "Green"
-Invoke-WebRequest -Uri "https://github.com/CharlieS1103/spicetify-marketplace/archive/refs/heads/main.zip" -UseBasicParsing -OutFile "${HOME}/spicetify-cli/CustomApps/spicetify-marketplace.zip"
+Invoke-WebRequest -Uri "https://github.com/spicetify/spicetify-marketplace/archive/refs/heads/dist.zip" -UseBasicParsing -OutFile "${HOME}/spicetify-cli/CustomApps/marketplace.zip"
 
 Write-Host "Unzipping and installing..." -ForegroundColor "Green"
-Expand-Archive -Path "${HOME}/spicetify-cli/CustomApps/spicetify-marketplace.zip" -DestinationPath "${HOME}/spicetify-cli/CustomApps/" -Force
-Remove-Item -Path "${HOME}/spicetify-cli/CustomApps/spicetify-marketplace.zip" -Force
-if (Test-Path -Path "${HOME}/spicetify-cli/CustomApps/spicetify-marketplace") {
-  Write-Host "spicetify-marketplace was already found! Updating..." -ForegroundColor "Cyan"
-  Remove-Item -Path "${HOME}/spicetify-cli/CustomApps/spicetify-marketplace" -Force -Recurse
+Expand-Archive -Path "${HOME}/spicetify-cli/CustomApps/marketplace.zip" -DestinationPath "${HOME}/spicetify-cli/CustomApps/" -Force
+Remove-Item -Path "${HOME}/spicetify-cli/CustomApps/marketplace.zip" -Force
+if (Test-Path -Path "${HOME}/spicetify-cli/CustomApps/marketplace") {
+  Write-Host "marketplace was already found! Updating..." -ForegroundColor "Cyan"
+  Remove-Item -Path "${HOME}/spicetify-cli/CustomApps/marketplace" -Force -Recurse
 }
-Rename-Item -Path "${HOME}/spicetify-cli/CustomApps/spicetify-marketplace-main" -NewName "spicetify-marketplace" -Force
-Copy-Item -Path "${HOME}/spicetify-cli/CustomApps/spicetify-marketplace" -Destination $sp_dot_dir -Recurse -Force
-spicetify config custom_apps spicetify-marketplace
+Rename-Item -Path "${HOME}/spicetify-cli/CustomApps/marketplace-dist" -NewName "marketplace" -Force
+Copy-Item -Path "${HOME}/spicetify-cli/CustomApps/marketplace" -Destination $sp_dot_dir -Recurse -Force
+spicetify config custom_apps marketplace
 spicetify backup apply
```

Comment:
```suggestion
spicetify backup
spicetify apply
```
Split commands as Spicetify would not let you apply if it is stacked right after `backup`, thus making the user manually applying again.

Reply:
Are you sure about this? I'm pretty sure it's worked in the past
ex: ```spicetify restore backup apply```

Reply:
It works with `spicetify restore backup apply` (the whole chain), not `spicetify backup apply` which would just stop at `backup`

Reply:
But using `restore backup apply` would take a significant amount of time plus most of the case Marketplace is installed right after the user installs Spicetify (bundled in docs) which is unnecessary.

Reply:
Maybe this is a bug in spicetify cli, but I've also had the issue detailed above.